spec/data/12098710-203-Patrice-shigazaru-upside-down.gtp

```diff
@@ -0,0 +1,207 @@
+
```

Comment:
Is this empty line part of the GTP spec?

Reply:
no, but it does not hurt. this extra space is cause to side effect of inheriting from SGF::Writer
maybe I should have reimplemented this different. or I could add a conditional

Reply:
removed